File: Java/PoaEShopping/src/org/pl/eshopping/aspects/admincategorias/AdminCategorias.java
package org.pl.eshopping.aspects.admincategorias;

import org.pl.eshopping.dto.Categoria;
import org.pl.eshopping.dto.CategoriaDAO;

public class AdminCategorias {
	
	public static CategoriaDAO cdao = new CategoriaDAO();
	
	public static void main(String[] args)
	{
		agregar("Ventanas","Ventanas de todos los tipos");
	}
	
	public static void agregar(String n, String d) {  
		Categoria c = new Categoria();
		c.setNombre(n);
		c.setDescripcion(d);
		cdao.agregar(c);
	}

}

File: PHP/PoaEShopping/aspects/admincategorias/AdminCategorias.php
<?php

namespace aspects\admincategorias;

include_once("../../config/db.php");
include_once("../../models/categoria.php");

class AdminCategorias{
	
	public function agregar($n, $d){
		$c=new \categoria();
		$c->setNombre($n);
		$c->setDescripcion($d);
		//$c->agregar();
	}
}
